virtcontainers/qemu.go

```diff
@@ -692,12 +701,16 @@ func (q *qemu) getMemArgs() (bool, string, string) {
 	} else if q.config.SharedFS == config.VirtioFS || q.config.FileBackedMemRootDir != "" {
 		target = q.qemuConfig.Memory.Path
 		memoryBack = "memory-backend-file"
+	} else if q.config.EnableVhostUserStore {
```

Comment:
Why is the `enable_vhost_user_store` hugepage test disabled if virtiofs is configured?

To me it should be:

```
if q.qemuConfig.Knobs.HugePages {
...
} else {
   if q.config.sharedFS ... { ... }
   if q.config.EnableVhostUserStore {  ... }
```

Or is it because you test it elsewhere?

Reply:
Ack. :)  Applied.
It was caused unintentionally by a previous manual rebase.